include/eve/arch/x86/predef.hpp

```diff
@@ -45,4 +45,17 @@
 #  define EVE_SUPPORTS_NATIVE_SIMD
 #  define EVE_HW_X86
 #  define EVE_INCLUDE_X86_HEADER
+
+// Don't trigger AVX512 if we don't have at least Skylake support
+#  if defined(SPY_SIMD_IS_X86_AVX512)
+#   if !defined(__AVX512BW__) && !defined(__AVX512CD__) \
+    && !defined(__AVX512DQ__) && !defined(__AVX512VL__)
```

Comment:
Docks say:
AVX512 with skylake like minimum flavour (f,vl,dq,bw)

this does not match.

SKYLAKE on AVX512 Wikipedia page says:

```
F, CD, ER, PF
VL, DQ, BW
```

Should this be the test? https://en.wikipedia.org/wiki/AVX-512

Update the doc maybe too?

Reply:
F is implied by IS_AVX86_AVX512. The ER and PF are indeed missing

Reply:
In fact, it is not. ER and PF are old Xeon Phi optiosn and are discontinued.
The if logic was wrong though and has been fixed